### ISSUE TITLE
Support for various global compliancies

### ISSUE THREAD

**Issue by jmakowski1123** [openedx/platform-roadmap#88]
Opportunity Objective
Open edX should be able to claim GDPR, FERPA, FDA 21p11, GDPR, WGAG, WAI, Section 508, ADA etc. 

this would put add Open edX points when choosing between it and other LMSes

Market Evidence
all of the above are requested compliancies from our customers. While in most cases we can explain what it takes to make Open edX compliant, it would help tremendously if some visible effort would be taken and then marketed in this regard.

Proposal Specifics
there need to be an investigation to be conducted for each potential compliancy, some plan elaborated and executed. as a result there will be some changes to the data-architecture and UI AND some FAQ and blog post should be created to show how Open edX is/can be compliant.

Success Measures
Open edX site/documentation clearly states selected comliancies

there's a visible proof for this claim (code, blogs, how-to's, etc)

Open edX is listed as XXX-comliant LMS in various LMS catalogs

[Source: https://openedx.atlassian.net/browse/OEROADMAP-62]

**Comment by jmakowski1123** [comment]
Requires splitting -
